internal/store/atlas/backup.go

```diff
@@ -60,3 +66,34 @@ func (s *Store) EnableEncryptionAtRest(projectID string) (*atlasv2.DataProtectio
 	}
 	return result, nil
 }
+
+func (s *Store) CreatePolicyItem(projectID string, policyItem *atlasv2.DiskBackupApiPolicyItem) (*atlasv2.DataProtectionSettings, error) {
+	compliancePolicy, _, err := s.clientv2.CloudBackupsApi.GetDataProtectionSettings(s.ctx, projectID).Execute()
+	if err != nil {
+		return nil, fmt.Errorf("couldn't create policy item: %w", err)
+	}
+
+	err = addItem(compliancePolicy, policyItem)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't create policy item: %w", err)
+	}
+
+	result, _, err := s.clientv2.CloudBackupsApi.UpdateDataProtectionSettings(s.ctx, projectID, compliancePolicy).Execute()
+	if err != nil {
+		return nil, fmt.Errorf("couldn't create policy item: %w", err)
+	}
+	return result, err
+}
+
+func addItem(compliancePolicy *atlasv2.DataProtectionSettings, item *atlasv2.DiskBackupApiPolicyItem) error {
+	if compliancePolicy == nil || item == nil {
+		return errors.New("either compliancePolicy or item is nil")
+	}
+	if item.FrequencyType == "ondemand" {
+		compliancePolicy.OnDemandPolicyItem = item
+	} else {
+		compliancePolicy.ScheduledPolicyItems = append(compliancePolicy.ScheduledPolicyItems, *item)
+	}
+
+	return nil
+}
```

Comment:
Looking at this makes me wonder why do we have a command that looks at two differen fields? this semes to me like we should have 
```
backups compliancepolicy policies ondemand [describe|create|update|delete]]
```

and 
```
backups compliancepolicy policies scheduled [list|describe|create|update|delete]]
```

Reply:
In the UI they are viewed together, in terms of the BCP struct/object/json (is there a generalized word for this?) the only difference is `frequencyType`. Of course, your proposal could be implemented and I can make a ticket for it, but I believe I don't have time to implement this as my internship is over in about 24 hours 🥲.

![Screenshot 2023-08-24 at 14 04 21](https://github.com/mongodb/mongodb-atlas-cli/assets/39012471/8fa459c2-35ed-4d04-bd5c-77b7d9d3db35)

Reply:
>is there a generalized word for this

Polymorphism

In general I don't think this command is right ly designed, for example I don't think it needed to be a file specially how is currently implemented, I think is ok if we pin policies for now as they may require further discussion